Shai solutions

ShaiGilboa opened the pull request.

ShaiGilboa committed:
part 1

ShaiGilboa committed:
workshop2 29/01

SSaquif commented:
Excellent job! pretty much perfect :) Only issue was the images for ex-7 would not render on my screen setting src='img/filename.ext" should solve this problem. Excellent work overall

ShaiGilboa committed:
second attempt, fixing the src for the imgs in 7

baased on the instruction, i've changed the path of the imgs to be the relative path "img/filename.jpeg" and not the full path "workshop/excercise7..../filename.jpeg"
I've checked it on for my and it seems ok.

ShaiGilboa commented:
I think it should work now.
